lib/matplotlib/tests/test_collections.py

```diff
@@ -325,6 +325,19 @@ def test__EventCollection__set_linestyle():
     splt.set_title('EventCollection: set_linestyle')
 
 
+@image_comparison(baseline_images=['EventCollection_plot__set_ls_dash'],
```

Comment:
It would be ok to test just one backend. I only want to test that the linestyle is set and interpreted correctly.
But I think that is true for a number of tests in this file...